Products/CMFPlone/_scripts/verifydb.py

```diff
@@ -77,11 +78,15 @@ def verify_record(oid, data, debug=False):
         ))
         logger.info(repr(pickle))
         logger.info(traceback.format_exc())
-        if debug:
+        if debug and pos is not None:
             try:
                 pickletools.dis(pickle[pos:])
+            except Exception:
+                logger.info(traceback.format_exc())
```

Comment:
Use logger.exception("some message" ) here.

Reply:
Not loging them as exceptions is kind of the point here to allow continuing to walk through all the issues in the DB while having a pdb for each.

Reply:
`logger.exception` does not raise, it just logs.

Reply:
Yes, but PDBDebugMode catches logger.exception which would be unwanted in thgis case.

Reply:
so, I would add a comment in there, Otherwise your future self (or mine or others) dont now about it.